docs/source/storage.rst

```diff
@@ -35,6 +35,61 @@ which is received by wis2box.
 Below are basic examples on sending data to the MinIO ``wis2box-incoming`` bucket.  For more information and additional
 examples, consult the `official MinIO documentation`_.
 
+
+Using the boto3 Python Client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install the Python boto3 package via `pip`_:
+
+.. code-block:: bash
+
+    pip3 install boto3
+
+The below example copies a local file (``myfile.csv``) to the ``wis2box-incoming`` bucket with topic ``foo.bar.baz``:
+
+
+.. code-block:: python
+
+    import boto3
+
+    endpoint_url = 'http://localhost:9000'
+    filename = 'myfile.csv'
+
+    session = boto3.Session(
+        aws_access_key_id='wis2box',
+        aws_secret_access_key='Wh00data!'
+    )
+
+    s3client = session.client('s3', endpoint_url=endpoint_url)
+
+    with open(filename, 'rb') as fh:
+        s3client.upload_fileobj(fh, 'wis2box-incoming', f'foo/bar/baz/{filename}')
+
+
+To allow uploading files into MinIO remotely, the ``wis2box-incoming`` bucket is proxied via Nginx. 
+
+For example, to upload the local file (``WIGOS_0-454-2-AWSNAMITAMBO_2021-11-18T0955.csv with topic``) with topic ``data.core.observations-surface-land.mw.FWCL.landFixed`` via the Nbinx proxy:
+
+.. code-block:: python
+
+    import boto3
+
+    endpoint_url = 'http://localhost:9000'
```

Comment:
Should this be `endpoint_url = $WIS2BOX_URL/wis2box-incoming`?

Reply:
wis2box-incoming is the bucket and is defined when calling upload_fileobj. I will replace this with <your-wis2box-url>